hawkular-alerts-rest/hawkular-alerts-rest-api/src/main/java/org/hawkular/alerts/rest/CrossTenantHandler.java

```diff
@@ -0,0 +1,328 @@
+                    allowableValues = "Timestamp in millisecond since epoch.")
+            @QueryParam("endTime")
+            final Long endTime,
+            @ApiParam(required = false, value = "Filter out alerts for unspecified alertIds.",
+                    allowableValues = "Comma separated list of alert IDs.")
+            @QueryParam("alertIds")
+            final String alertIds,
+            @ApiParam(required = false, value = "Filter out alerts for unspecified triggers. ",
+                    allowableValues = "Comma separated list of trigger IDs.")
+            @QueryParam("triggerIds")
+            final String triggerIds,
+            @ApiParam(required = false, value = "Filter out alerts for unspecified lifecycle status.",
+                    allowableValues = "Comma separated list of [OPEN, ACKNOWLEDGED, RESOLVED]")
+            @QueryParam("statuses")
+            final String statuses,
+            @ApiParam(required = false, value = "Filter out alerts for unspecified severity. ",
+                    allowableValues = "Comma separated list of [LOW, MEDIUM, HIGH, CRITICAL]")
+            @QueryParam("severities")
+            final String severities,
+            @ApiParam(required = false, value = "Filter out events for unspecified tags.",
+                    allowableValues = "Comma separated list of tags, each tag of format 'name\\|value'. + \n" +
+                            "Specify '*' for value to match all values.")
+            @QueryParam("tags")
+            final String tags,
+            @ApiParam(required = false, value = "Filter out alerts resolved before this time.",
+                    allowableValues = "Timestamp in millisecond since epoch.")
+            @QueryParam("startResolvedTime")
+            final Long startResolvedTime,
+            @ApiParam(required = false, value = "Filter out alerts resolved after this time.",
+                    allowableValues = "Timestamp in millisecond since epoch.")
+            @QueryParam("endResolvedTime")
+            final Long endResolvedTime,
+            @ApiParam(required = false, value = "Filter out alerts acknowledged before this time.",
+                    allowableValues = "Timestamp in millisecond since epoch.")
+            @QueryParam("startAckTime")
+            final Long startAckTime,
+            @ApiParam(required = false, value = "Filter out alerts acknowledged after this time.",
+                    allowableValues = "Timestamp in millisecond since epoch.")
+            @QueryParam("endAckTime")
+            final Long endAckTime,
+            @ApiParam(required = false, value = "Return only thin alerts, do not include: evalSets, resolvedEvalSets.")
+            @QueryParam("thin")
+            final Boolean thin,
+            @Context
+            final UriInfo uri) {
+        Pager pager = RequestUtil.extractPaging(uri);
+        try {
+            Set<String> tenantIds = getTenants(tenantId);
```

Comment:
Yes.
In theory, it something defined at HTTP standard:
https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2

But +1 I will add some note to document that a comma list of tenants are necessary to use these endpoints.